creator/buckets/scanner.py

```diff
@@ -33,6 +34,7 @@ def sync_buckets():
             bucket = Bucket(
                 name=bucket_info["Name"],
                 created_on=bucket_info["CreationDate"],
+                organization=Organization.objects.earliest("created_on"),
```

Comment:
My plan is to create the `Default Organization` but then rename it to `Kids First DRC` in production. This will break any query we make for `Default Organization` not to mention anyone may make a second org called `Default Organization`. I think that simply pulling out the first created org is pretty reliable for getting the organization we want (Kids First).

Reply:
Actually why don't we name the default organization `Kids First DRC` right from the start?

Reply:
Hmm should organization name be unique then?

Reply:
We don't name it 'Kids First DRC' in the migration because you may imagine a time when we want to make a new deployment of the Data Tracker where that name may be confusing. It's just as easy to name it something generic then use one of the mutations to update it to something that fits the needs of the deployment after.

Reply:
We could make organization names unique if we want to instead slugify it and provide urls with the org name. This would be a bit easier to share urls around, but then it may be messy if we wanted to rename one of the organizations as it may break links.